string.c

```diff
@@ -11475,12 +11479,13 @@ rb_sym_proc_call(ID mid, int argc, const VALUE *argv, int kw_splat, VALUE passed
 
 #if 0
 /*
- * call-seq:
- *   sym.to_proc
+ *  call-seq:
+ *    to_proc
+ *
+ *  Returns a Proc object corresponding to +self+:
  *
- * Returns a _Proc_ object which responds to the given method by _sym_.
+ *    :foo.to_proc # => #<Proc:0x000001afe0e48680(&:foo) (lambda)>
  *
- *   (1..3).collect(&:to_s)  #=> ["1", "2", "3"]
```

Comment:
I think it's worth mentioning the `&:` syntax.

Reply:
Done.